core/test/uix/aot_test.clj

```diff
@@ -4,6 +4,14 @@
             [uix.compiler.attributes :as attrs]))
 
 (deftest test-parse-tag
+  (is (= (attrs/parse-tag (name :#id.class))
+         ["div" "id" "class" false]))
+  (is (= (attrs/parse-tag (name :.class))
+         ["div" nil "class" false]))
+  (is (= (attrs/parse-tag (name :#id))
+         ["div" "id" nil false]))
+  (is (= (attrs/parse-tag (name :div.class#id.klass))
+         ["div" "id" "class klass" false]))
```

Comment:
I am not sure about this one. Maybe the id should always come first? I personally feel like this might be more confusing than valuable...